react/features/speaker-stats/functions.js

```diff
@@ -63,56 +62,66 @@ export function getPendingReorder(state: Object) {
 }
 
 /**
- * Get sorted speaker stats based on a configuration setting.
+ * Get sorted speaker stats ids based on a configuration setting.
  *
  * @param {Object} state - The redux state.
  * @param {Object} stats - The current speaker stats.
- * @returns {Object} - Ordered speaker stats.
+ * @returns {Object} - Ordered speaker stats ids.
  * @public
  */
-export function getSortedSpeakerStats(state: Object, stats: Object) {
+export function getSortedSpeakerStatsIds(state: Object, stats: Object) {
```

Comment:
Instead of returning a sorted object of speaker stats, we are following the same sorting logic but just returning an array of participant ids instead.